tests/lms/resources/_lti_launch_test.py

```diff
@@ -269,45 +269,26 @@ def test_h_username_raises_if_tool_consumer_instance_guid_is_missing(
             HTTPBadRequest,
             match='Required parameter "tool_consumer_instance_guid" missing from LTI params',
         ):
-            resources.LTILaunchResource(pyramid_request).h_username
+            resources.LTILaunchResource(pyramid_request).h_user
 
-    def test_h_username_raises_if_user_id_is_missing(self, pyramid_request):
+    def test_h_user_raises_if_user_id_is_missing(self, pyramid_request):
         pyramid_request.params = {"tool_consumer_instance_guid": "VCSy*G1u3:canvas-lms"}
 
         with pytest.raises(
             HTTPBadRequest, match='Required parameter "user_id" missing from LTI params'
         ):
-            resources.LTILaunchResource(pyramid_request).h_username
+            resources.LTILaunchResource(pyramid_request).h_user
 
-    def test_h_userid(self, pyramid_request):
+    def test_h_user_userid(self, pyramid_request):
         pyramid_request.params = {
             "tool_consumer_instance_guid": "VCSy*G1u3:canvas-lms",
             "user_id": "4533***70d9",
         }
 
-        userid = resources.LTILaunchResource(pyramid_request).h_userid
+        userid = resources.LTILaunchResource(pyramid_request).h_user.userid
 
         assert userid == "acct:2569ad7b99f316ecc7dfee5c0c801c@TEST_AUTHORITY"
 
-    def test_h_userid_raises_if_tool_consumer_instance_guid_is_missing(
```

Comment:
After combining the properties, these tests become unnecessary because they are already covered by the `h_user` tests.